File: AndroidStudio/ViewPageCustom/app/src/main/java/to/msn/wings/viewpagecustom/PageAdapter.java
package to.msn.wings.viewpagecustom;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentPagerAdapter;

public class PageAdapter extends FragmentPagerAdapter {
        //ページ数
    private final int PAGE_COUNT = 3;
        //ページのタイトル
    private String[] titles = new String[] {"ゾウ", "キャラクター", "アザラシ"};

    PageAdapter(FragmentManager fm){
        super(fm);
    }

        //インデックス番号に応じてページを取得
    @NonNull
    @Override
    public Fragment getItem(int index) {
         switch (index) {
             case 0:
                 return new Page1Fragment();
             case 1:
                 return new Page2Fragment();
             default:
                 return new Page3Fragment();
         }
    }
        //ページの個数を取得
    @Override
    public int getCount() {
        return PAGE_COUNT;
    }

        //ページのタイトルを取得
    @Override
    public CharSequence getPageTitle(int index){
        return titles[index];
    }
}

File: Android_MyApp/ImpartialReferee/app/src/main/java/to/msn/wings/impartialreferee/PageAdapter.kt
package to.msn.wings.impartialreferee

import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentManager
import androidx.fragment.app.FragmentPagerAdapter

/**
 * リストを表示する為のアダプタークラス
 */
class PageAdapter(fm:FragmentManager) : FragmentPagerAdapter(fm){
    //ページ数
    private val PAGE_COUNT = 1
    //ページのタイトル
    private var titles:Array<String> = arrayOf("ページ1")

    //================================================================================//

    /**
     * インデックス番号に応じてページを取得
     */
    override fun getItem(position: Int): Fragment {
        return ListFragment1()
    }



    /**
     * ページ数を返す
     */
    override fun getCount(): Int {
       return PAGE_COUNT
    }
}